source/store/AuthContext.js

```diff
@@ -51,17 +62,17 @@ function AuthProvider({ children, initialState }) {
   async function handleAuth(personalNumber, authenticateOnExternalDevice) {
     // Dynamically sets app in dev mode
     if (personalNumber && env.TEST_PERSONAL_NUMBER === personalNumber) {
-      handleSetMode('development');
+      handleSetMode("development");
     }
 
     // Disable BankId authentication
```

Comment:
Maybe we could clean noise like this as we do other refactoring?

Reply:
Perhaps, but its out of scope in my PR